src/scss/index.scss

```diff
@@ -8,17 +8,16 @@
 
 // Pages
 @import './custom/customFile';
-@import "./custom/createOffer";
+@import './custom/createOffer';
 @import './custom/allListingPage';
 
 @import './custom/admin';
 
-
 body {
-    height: 100vh;
-    display: grid;
-    grid-template-rows: auto 1fr auto;
+  height: 100vh;
+  display: grid;
+  grid-template-rows: auto 1fr auto;
 }
```

Comment:
I would move this down under the import of bootstrap

Reply:
See comment below.